content-services/content-sources-frontend
# Fixes 2048: add without snapshot

## Summary

Changes add and bulk add popular repository buttons to include a dropdown when the snapshot feature is enabled. Snapshots will be enabled by default, but opening the dropdown will have a button that allows adding popular repositories without snapshotting. When snapshotting is disabled, the buttons will appear the same as before.

## Testing steps

Navigate to `beta/settings/content/popular-repositories`

Ensure that when snapshots are enabled, clicking the add and bulk add buttons create repositories with snapshot = True. Clicking the "Add without snapshotting" buttons should add snapshot = False. The buttons should still be disabled when rbac.write = False.

Test that when snapshotting is disabled or not accessible for the current user that clicking the add buttons should create repositories with snapshot = False.
